lib/pure/parsecfg.nim

```diff
@@ -482,54 +494,62 @@ proc replace(s: string): string =
     inc(i)
   result = d
 
+proc writeConfigToStream*(dict: Config, stream: Stream) =
+  ## Writes the contents of the table to the specified stream
+  ## Note: Comment statement will be ignored.
+  var section, key, value, kv, segmentChar:string
```

Comment:
It might be a good idea to split this function up into other smaller private functions.